test/doc/algo/find.cpp

```diff
@@ -6,20 +6,31 @@
 
 int main()
 {
```

Comment:
I would prefer to have a separate .cpp for backward as if not, the doc is a bit loong to read.